### PR TITLE
Removed Web Setup Wizard Content (install)

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) removes references to the Web Setup Wizard as it relates to installation for 2.3.x. 

It addresses changes related to installation only. I will address changes related to upgrade and extension management in subsequent PRs because this PR is already so large.

Both passes will be merged with the `MC-40790-remove-wizard-integration` branch before being merged with the `2.3.7-develop` branch.

- Removed 2.3 topics in Setup > Installation Guide > Setup Wizard Install
- Redirected 2.3 topics to 2.3 CLI install topics
- Removed dedicated 2.3 TOC
   - I simply updated the `_config.yml`, but I think the right thing to do is to add the group to the metadata of each file in the install guide. @dshevtsov, what do you recommend? 
- Re-established symlinks between 2.3 and 2.4 files (reverting work done in #7246)
  - File diffs proved that the only diff was related to the wizard
- Removed redundant includes for 2.3 and 2.4 files (reverting work done in #7246)
   - File diffs proved that the only diff was related to the wizard
- Removed deprecation messaging
- Aligned 2.3 install guide with recent 2.4 optimizations

## Affected DevDocs pages

The primary area of focus should be on the pages inside the Setup Wizard Install section of the Installation Guide. These topics should appear in 2.3, but should redirect to the CLI install topics in 2.4:

Screen Shot 2020-05-19 at 4 54 01 PM

## Additional info

See internal ticket MC-40790 for more info and link to staging build (`#168`).

### CODE REVIEW

File: src/guides/v2.3/install-gde/bk-install-guide.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ functional_areas:
   - Install
   - System
   - Setup
+redirect_from:
+  - /guides/v2.4/install-gde/continue-to-install.html
```
Comment (dobooth) [inline src/guides/v2.3/install-gde/bk-install-guide.md:9]: I don't understand this redirect. I don't usually see a 2.4-> 2.3 version change in these.

File: src/guides/v2.3/install-gde/bk-install-guide.md [hunk 2]
```diff
@@ -13,12 +15,10 @@ We have some resources here to help get you started using the eCommerce platform
 
 ## How to get the Magento software {#install-get-software}
 
-Check the availability of exciting new features and releases, and learn how you can obtain them, on our [Magento 2.3 product availability page](https://devdocs.magento.com/release/availability.html).
+Check the availability of exciting new features and releases, and learn how you can obtain them, on our [Magento product availability page](https://devdocs.magento.com/release/availability.html).
```
Comment (dobooth) [inline src/guides/v2.3/install-gde/bk-install-guide.md:18]: A nit pick for sure, but can we get rid of `exciting`?

**Review by dobooth — APPROVED** [review]
Excellent work!

File: src/guides/v2.3/install-gde/bk-install-guide.md (hunk 1 above)
Comment (jeff-matthews) [reply src/guides/v2.3/install-gde/bk-install-guide.md:9]: Good catch!

File: src/guides/v2.3/install-gde/bk-install-guide.md (hunk 2 above)
Comment (jeff-matthews) [reply src/guides/v2.3/install-gde/bk-install-guide.md:18]: No problem

**Pull request merged**